draft-packages/stories/IllustrationScene.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react"
```

Comment:
Not for this PR - but we should look at a page on the site that lists all the icons

Reply:
Mitigated by having a story for every illustration